requirements/system-requirements.md

```diff
@@ -29,19 +29,20 @@ This article lists the minimum hardware and software requirements for the instal
 
 The application can be installed on a workstation with __Windows 10, Windows 8 or Windows 7 operating system__, but you will not be able to load new SharePoint farm settings, but only connect to an existing SPDocKit database and open already saved farm settings.
 
-From the version 5.2., SPDocKit installed on a workstation also supports connecting to any __SharePoint 2010, SharePoint 2013, SharePoint 2016 or SharePoint Online__ site and real-time viewing and management of permissions. 
+From the version 5.2., SPDocKit installed on a workstation also supports connecting to any __SharePoint 2010, SharePoint 2013, SharePoint 2016__ site and real-time viewing and management of permissions. 
 
 * Hardware:
   * CPU – any Windows 7, Windows 8 or Windows 10 capable CPU
   * Memory – up to 200 MB RAM while idle
   * Disk – 100 MB disk space
 
-### Running SPDocKit Event Collection Service
+### Running SPDocKit Insights Agent
 
-From the version 7.0, SPDocKit offers the new installation of __SPDocKit Insights Agent__. It is used for real-time collection and searching of the ULS logs from SharePoint 2010, SharePoint 2013, or SharePoint 2016.
+From the version 7.0, SPDocKit offers the new installation of __SPDocKit Insights Agent__. It is used for performance monitoring and real-time collection and searching of the ULS logs from SharePoint 2010, SharePoint 2013, or SharePoint 2016.
```

Comment:
From version 7.0, SPDocKit offers the new installation of __SPDocKit Insights Agent__. It is used for performance monitoring and real-time collection and searching of ULS logs from SharePoint 2010, SharePoint 2013, or SharePoint 2016. [articles too many]